pos_barcode_sync/models/pos_sync_barcodes_models.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api
+from odoo.http import request
+
+
+def send_barcode_update(message, partner_ids):
+    channel_name = "pos_barcode_sync"
+    pos_configs = request.env["pos.config"].search([('active', '=', True)])
+    res = pos_configs._send_to_channel(channel_name, {'message': message, 'partner_ids': partner_ids})
+    return res
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def write(self, vals):
+        result = super(ResPartner, self).write(vals)
+        send_barcode_update('Update Partner Barcode', self.ids)
```

Comment:
It's a bad idea to use space and capitals in the technical message. Because For example, it's easy to make mistake ( miss capitals or use two spaces instead of one).

I know, that this message is not used now, but it's supposed to be used in later versions